test/jdk/javax/swing/JEditorPane/4694598/FrameContent.html

```diff
@@ -0,0 +1,3 @@
+<HTML><BODY>
```

Comment:
Looks like you forgot to remove this file ?

Reply:
I'm also a bit surprised I don't see all the rest of the original html & java files anywhere now, not as deleted, not as changed.

Reply:
There's something wrong with the latest commit. Looking at it now.

Reply:
Ok, either github or skara got mad at me for pushing commit with both rename and edit changes of the same files. Tried to fix it but it only gets worse so what i'm going to do is i will clone a fresh repository, move stuff there, close this PR and create a new one tomorrow morning with the latest versions of files in correct places.